lib/npm-auditer.ts

```diff
@@ -71,18 +79,40 @@ function printReport(
       printReportObject("NPM audit report JSON:", parsedOutput);
       break;
     case "important": {
-      const advisories = isV2Audit(parsedOutput)
-        ? parsedOutput.vulnerabilities
-        : parsedOutput.advisories;
+      const relevantAdvisories = (() => {
```

Comment:
Functionally, this is equivalent to the previous definition. However, this one is much more type-safe.